libraries/importers/component-importer/src/main/java/org/eclipse/sw360/importer/ComponentImportUtils.java

```diff
@@ -469,9 +469,21 @@ public ReleaseLinkCSVRecord apply(CSVRecord input) {
     private static Map<String, String> getVendorNameToVendorId(VendorService.Iface vendorClient) throws TException {
         Map<String, String> vendorNameToVendorId = new HashMap<>();
 
-        for (Vendor vendor : vendorClient.getAllVendors()) {
-            vendorNameToVendorId.put(vendor.getFullname(), vendor.getId());
-        }
+            for (Vendor vendor : vendorClient.getAllVendors()) {
+                if (!vendorNameToVendorId.containsKey(vendor.getShortname())) {
+                    vendorNameToVendorId.put(vendor.getShortname(), vendor.getId());
+                }
+                else {
```

Comment:
That's an unusual formatting. The convention is to place `else` on the same line as the previous closing curly brace. Like this: `} else {`.
Same goes for line 482